mcache/tests/test_check.py

```diff
@@ -216,6 +269,23 @@ def test_service_ok(check, instance, aggregator, memcached):
     assert sc.tags == tags
 
 
+# This test doesn't work with Mac either.
+# See https://docs.docker.com/docker-for-mac/osxfs/#file-types
+# See open PR https://github.com/docker/for-mac/issues/483
+@pytest.mark.skipif(not Platform.is_linux(), reason='Windows sockets are not file handles')
+def test_service_with_socket_ok(check, instance_socket, aggregator, memcached_socket):
```

Comment:
Finally, could we separate unit and integration test using pytest marks, and create two tox environments please ?

Reply:
Actually I think this is unnecessary since we only really have integration tests here. Also, I'm not a fan of the distinction in general for us.

Reply:
I will leave it as is since we only have integration tests here

Reply:
Well there are several tests that don't use the `memcached` and `memcached_socket` fixtures. Not sure if we can call these integration tests. 
Why are you not a fan of the distinction ? In general, it allows a user to iterate faster on unit tests by not spinning up the docker containers, since they can run a specific environment with `tox -e`.

Reply:
both fixtures are for integration tests. I may be wrong but I believe a fixture is only used if it is added as a param into the test method.

Reply:
Yeah that's what I meant. If a test function is not using the fixture `memcached` nor `memcached_socket` then it is not an integration test IMO. That is why I said that there is a mix of both unit and integration tests in here.

Reply:
Yeah! you are actually right, there are some tests that are not integration tests. I will split them then